importers/ableroad.com/easy_importer.py

```diff
@@ -0,0 +1,226 @@
+#!/home/liu1ee/apps/virtualenv/bin/python
+from lxml import html
+import requests
+import os.path
+import re
+import csv
+import urllib2
+import logging
+import time
+
+def getlatlng(js):
+   """
+   a sample js string lookes like this
+   #//debug stuff maphidden = 0;
+       if (!maphidden) {
+                createSearchMarker('20. Craft Heads Brewing Company','<label style="display:none">phone and address</label>89 University Avenue W<br/>Windsor, ON N9A 5N8<br/>Canada<br/>+1-226-246-3925','','ableroad.com/edit.php?index=20&amp;newID=craft-heads-brewing-company-windsor&amp;s=&amp;s1=windsor,ontario&amp;action=write',42.317252218728,-83.039797096553,20 - 1,1);
+                     }
+   """
+   js = js.encode('ascii','ignore') # unicode to ascii
```

Comment:
Always add a space after a comma as a general rule to stay consistent with how the code looks.

Search for commas in your new files and make sure there is 1 regular space after each one.